bot/exts/events/trivianight/trivianight.py

```diff
@@ -0,0 +1,202 @@
+                description=(
+                    "This 'cog' is for the Python Discord's TriviaNight (date tentative)! Compete against other"
+                    "players in a trivia about Python!"
+                ),
+                color=Colours.soft_green
+            )
+            await ctx.send(embed=cog_description)
+
+    @trivianight.command()
+    async def load(self, ctx: commands.Context, *, to_load: Optional[str]) -> None:
+        """
+        Loads a JSON file from the provided attachment or argument.
+
+        The JSON provided is formatted where it is a list of dictionaries, each dictionary containing the keys below:
+        - number: int (represents the current question #)
+        - description: str (represents the question itself)
+        - answers: list (represents the different answers possible, must be a length of 4)
+        - correct: str (represents the correct answer in terms of what the correct answer is in `answers`
+        - time: Optional[int] (represents the timer for the question and how long it should run, default is 10)
+        - points: Optional[int] (represents how many points are awarded for each question, default is 10)
+
+        The load command accepts three different ways of loading in a JSON:
+        - an attachment of the JSON file
+        - a message link to the attachment/JSON
+        - reading the JSON itself via a codeblock or plain text
+        """
+        if ctx.message.attachments:
+            json_text = (await ctx.message.attachments[0].read()).decode("utf8")
+        elif to_load.startswith("https://discord.com/channels") or \
+                to_load.startswith("https://discordapp.com/channels"):
+            channel_id, message_id = to_load.split("/")[-2:]
+            channel = await ctx.guild.fetch_channel(int(channel_id))
+            message = await channel.fetch_message(int(message_id))
+            if message.attachments:
+                json_text = (await message.attachments[0].read()).decode("utf8")
+            else:
+                json_text = message.content.replace("```", "").replace("json", "").replace("\n", "")
+
+        try:
+            serialized_json = loads(json_text)
+        except JSONDecodeError:
+            raise commands.BadArgument("Invalid JSON")
+
+        for idx, question in enumerate(serialized_json):
+            serialized_json[idx] = {**question, **{"description": self.unicodeify(question["description"])}}
+        self.questions.view = QuestionView()
+        self.scoreboard.view = ScoreboardView(self.bot)
+        self.questions.set_questions(serialized_json)
+        success_embed = Embed(
+            title=choice(POSITIVE_REPLIES),
+            description="The JSON was loaded successfully!",
+            color=Colours.soft_green
+        )
+        await ctx.send(embed=success_embed)
+
+    @trivianight.command()
+    async def reset(self, ctx: commands.Context) -> None:
+        """Resets previous questions and scoreboards."""
```

Comment:
This is meant to be used after the game right?

Reply:
I see your thumbs up, one thing I like to do in my code is to have the reaet logic be shared with the initializing code.

This way, you dont forget to reset anything 😉. So my recommendation would be to have like a `setup()` or `initialize()` method that is called by both the `reset` command and `__init__()`.

Reply:
Gotcha! I'll add this after updating the command checks